test/versioned/winston/winston.tap.js

```diff
@@ -48,38 +48,75 @@ tap.test('Winston instrumentation', { bail: true }, (t) => {
     }
   }
 
-  const logStuff = ({ logger, stream }) => {
-    // Log some stuff, both in and out of a transaction
-    logger.info('out of trans')
+  /**
+   * Log lines in and out of a transaction for eveyr logger.
+   * @param {Object} opts
+   * @param {DerivedLogger} opts.logger instance of winston
+   * @param {Array} opts.loggers an array of winston loggers
+   * @param {Stream} opts.stream stream used to end test
+   * @param {Test} opts.t tap test
```

Comment:
According to this docstring, shouldn't we also destructure `t`?

Reply:
derp, copy pasta fail. this function doesn't use the test to assert anything